### DISCUSSION TITLE
Getting rid of indentations for manim im jupyter

### DISCUSSION THREAD

**Discussion by kolibril13** [ManimCommunity/manim#2794] [Suggestions and Proposals]
It would be great to have something like this in Jupyter:
```py
scn = Scene()
dORIGIN= Dot(color= BLUE,radius=0.5)
d= Dot(color= YELLOW, radius=0.5)
d.shift(2*RIGHT)
scn.add(d)
scn.render()
```
Similar to how Matplotlib plots are rendered.
This is how it could look then compared to the current way (not yet working):
![image](https://user-images.githubusercontent.com/44469195/122522988-d406b780-d016-11eb-8612-2580010d51ab.png)

**Comment by behackl** [comment]
Allowing syntax like that is nothing that is specific to Jupyter.

**Comment by hydrobeam** [comment]
This idea looks similar to [manim-express](https://github.com/beidongjiedeguang/manim-express) which someone made for use with manimgl . I think not being forced into two levels of indentation would be a nice idea and would make drafting quick scenes easier.

**Comment by k4pran** [comment]
It seems that outside of jupyter it renders find, but just gives the error

```
                    ERROR                                       module_ops.py:72
                                There are no scenes inside that                 
                             module 
```

Is it just this error and special handling for jupyter that is needed?
